server/app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+const routerTodo = require('./routes/todos.js')
+const app = express()
+// const bodyParser = require('body-parser')
+const port = process.env.port || 3000
```

Comment:
biasakan pakai PORT yah , ini nanti berguna untuk kebutuhan deploy
```suggestion
const port = process.env.PORT || 3000
```